packages/manager/core/generator/app/conditional-templates/listing/index-api-v2-pagination-cursor.tsx.hbs

```diff
@@ -0,0 +1,70 @@
+import React, { useState } from 'react';
+import { Navigate } from 'react-router-dom';
+import { useInfiniteQuery } from '@tanstack/react-query';
+import { OsdsButton } from '@ovhcloud/ods-components/button/react';
+import { ODS_BUTTON_VARIANT } from '@ovhcloud/ods-components/button';
+import { ODS_THEME_COLOR_INTENT } from '@ovhcloud/ods-common-theming';
+import { getListingIcebergV2 } from '@/api';
+
+import Datagrid from '@/components/layout-helpers/Listing/dataGrid';
+import Loading from '@/components/Loading/Loading';
+import ErrorBanner from '@/components/Error/Error';
+import appConfig from '@/{{this.appName}}.config';
+import Breadcrumb, {
+  BreadcrumbHandleParams,
+} from '@/components/Breadcrumb/Breadcrumb';
+
+export function breadcrumb({ params }: BreadcrumbHandleParams) {
+  return params.serviceName;
+}
+
+
+export default function Listing() {
+  const [pageSize] = useState(10);
+
+  const { data, fetchNextPage, hasNextPage, isError, isLoading, error, }: any = useInfiniteQuery({
+    initialPageParam: null,
+    queryKey: [`servicesListingIceberg`],
```

Comment:
The querykey should be exported somewhere to be able to reuse the query result on multiple pages